docs/release/handbook.md

```diff
@@ -0,0 +1,288 @@
+# Kubeflow Release Handbook
+
+## Executive Summary
+
+The purpose of this document is to define the roles and processes for the Kubeflow community to release Kubeflow. This document will attempt to coordinate releasing between the five major components and working groups in Kubeflow, as well as defining best practices, roles, and processes to make releasing Kubeflow easier for the community with each iteration.
+
+## People and Roles
+
+### Release Manager
+The Release Manager will be responsible for coordinating the release and taking ultimate accountability for all release tasks to be completed on time.
+
+**Prerequisites:**
+1. Be a contributor for at least 2 years
+2. Have shadowed in a previous release or be a Release Team member in the past
```

Comment:
There is a word here "shadowed" and in the terminology section "shadowing". However, I don't see that role in the "people and roles" section. Could we either remove this or add a new role called "release manager shadow" and define their prerequisite and responsibilities?

Reply:
Given that Kubeflow has only been around for 3 years, I believe that restricting to 2 years of contributions is not very realistic at this point. I also agree with the previous comments from @mkbhanda that restricting it to 2 years of contributing be too large of a hurdle for new people involved in the project out of passion and curiosity (at least, that is how I interpreted it).

Reply:
I believe that these pre-requisites are too constraining for capable and available community members to contribute as future leads. 

I would add that, if the community agrees on a Release Manager as opposed to Release Team based decisions, there should be a change in the leading organization (i.e. organization that employs Release Manager) on a rotation basis. I would avoid a situation where releases are constantly lead by one organization in the presence of a capable community.

Reply:
Indeed, the initial requirements are kind of restrictive. 

I'm OK with even removing all requirements for the team members, in order to allow people who are passionate about the project to also help in the release process.

Regarding the manager the reasoning behind the 2 years of contributions is to ensure that the release manager will be a person that is heavily invested into the project. Although it also makes sense to drop the time requirement to 1 year.

Reply:
Added a commit that:
1. Rephrases the `Shadow` terminology
2. Reduced the requirements for the manager to 1 year

So since the discussion points were addressed I'll move forward and resolve this conversation